src/web/lib/decidePalette.ts

```diff
@@ -624,6 +628,8 @@ const borderNavPillar: (format: Format) => string = (format) =>
 	pillarPalette[format.theme].bright;
 
 const borderArticle: (format: Format) => string = (format) => {
+	if (format.design === Design.LiveBlog || format.design === Design.DeadBlog)
+		return '#CDCDCD';
```

Comment:
Again, more a future question cc @HarryFischer is it possible to use a Source/palette colour here. (Or if this exact shade required, should we add this to Source?)

Reply:
(And for the other border colours we use.)

Reply:
Yeah, there's a few of these where you really need the exact shade for the border to look right and Source doesn't have them. It brings us back to the idea of grouping colours. Each backgrond colour could have it's own border and text variations

Reply:
Leaving these as hex strings for now as per the the conversation irl with @HarryFischer . I agree that they should ideally be values from Source but I think given that they're already abstracted behind `palette` there's less impact compared to just randomly dropping hex values in the code.